runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientConnectionPoolFactory.java

```diff
@@ -0,0 +1,1095 @@
+
+        sender.accept(reset.typeId(), reset.buffer(), reset.offset(), reset.sizeof());
+    }
+
+    final class KafkaClientConnectionApp
+    {
+        private final long originId;
+        private final long routedId;
+        private final long authorization;
+        private final String host;
+        private final int port;
+        private final KafkaClientConnectionNet connection;
+        private final Long2ObjectHashMap<MessageConsumer> senders;
+
+        private long initialSeq;
+        private long initialAck;
+        private int initialMax;
+        private int initialPad;
+        private long initialBud;
+
+        private long replySeq;
+        private long replyAck;
+        private int replyMax;
+        private int replyPad;
+        private long replyBud;
+
+        private long reconnectAt = NO_CANCEL_ID;
+
+        private KafkaClientConnectionApp(
+            long originId,
+            long routedId,
+            long initialId,
+            long authorization,
+            long initialBud,
+            String host,
+            int port)
+        {
+            this.originId = originId;
+            this.routedId = routedId;
+            this.authorization = authorization;
+            this.initialBud = initialBud;
+            this.host = host;
+            this.port = port;
+            this.senders = new Long2ObjectHashMap();
+            this.connection = new KafkaClientConnectionNet(this, originId, routedId, initialId, authorization);
+        }
+
+        private void onConnectionMessage(
+            int msgTypeId,
+            DirectBuffer buffer,
+            int index,
+            int length)
+        {
+            switch (msgTypeId)
+            {
+            case BeginFW.TYPE_ID:
+                final BeginFW begin = beginRO.wrap(buffer, index, index + length);
+                onConnectionInitialBegin(begin);
+                break;
+            case DataFW.TYPE_ID:
+                final DataFW data = dataRO.wrap(buffer, index, index + length);
+                onConnectionInitialData(data);
+                break;
+            case EndFW.TYPE_ID:
+                final EndFW end = endRO.wrap(buffer, index, index + length);
+                onConnectionInitialEnd(end);
+                break;
+            case AbortFW.TYPE_ID:
+                final AbortFW abort = abortRO.wrap(buffer, index, index + length);
+                onConnectionInitialAbort(abort);
+                break;
+            case WindowFW.TYPE_ID:
+                final WindowFW window = windowRO.wrap(buffer, index, index + length);
+                onConnectionReplyWindow(window);
+                break;
+            case ResetFW.TYPE_ID:
+                final ResetFW reset = resetRO.wrap(buffer, index, index + length);
+                onConnectionReplyReset(reset);
+                break;
+            default:
+                break;
+            }
+        }
+
+        private void onConnectionInitialBegin(
```

Comment:
Please remove `Initial` and `Reply` from method names, that is old style before we unified on single `on...Message` callback for both `initial` stream and `reply` throttle.